### PR TITLE
Bump Iglu Server to 0.4.0

### PR DESCRIPTION
**Pull request by oguzhanunlu**
Latest Iglu release, R10 Tiflis, introduced Iglu Server `0.4.0`.

### CODE REVIEW

File: iglu-server/0.4.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM snowplow-docker-registry.bintray.io/snowplow/base-alpine:0.2.0
```
Comment (BenFradet) [inline iglu-server/0.4.0/Dockerfile]: since we might want to deploy that in gcp too we might be better off with debian

Comment (BenFradet) [reply iglu-server/0.4.0/Dockerfile]: be aware that this means gosu and not su-exec, and not apk

**Comment by chuwy** [comment]
Hey @BenFradet, I've addressed your feedback. Could you please take another look.

**Review by BenFradet — APPROVED** [review]
Thanks Anton!

**Pull request merged**